airbyte-server/src/main/java/io/airbyte/server/EventListener.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server;
+
+import io.airbyte.commons.lang.CloseableShutdownHook;
+import io.airbyte.config.Configs;
+import io.airbyte.config.EnvConfigs;
+import io.airbyte.db.factory.DataSourceFactory;
+import io.airbyte.db.factory.FlywayFactory;
+import io.airbyte.db.instance.configs.ConfigsDatabaseMigrator;
+import io.airbyte.db.instance.jobs.JobsDatabaseMigrator;
+import io.airbyte.server.handlers.DbMigrationHandler;
+import io.micronaut.runtime.event.ApplicationStartupEvent;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+import jakarta.inject.Singleton;
+import java.sql.Connection;
+import javax.sql.DataSource;
+import lombok.extern.slf4j.Slf4j;
+import org.flywaydb.core.Flyway;
+import org.jooq.DSLContext;
+import org.jooq.SQLDialect;
+import org.jooq.impl.DSL;
+
+@Singleton
+@Slf4j
+public class EventListener {
+
+  private static final String DRIVER_CLASS_NAME = "org.postgresql.Driver";
+
+  @io.micronaut.runtime.event.annotation.EventListener
+  @ExecuteOn(TaskExecutors.IO)
+  @SuppressWarnings({"PMD.AvoidCatchingThrowable", "PMD.DoNotTerminateVM"})
+  public void startEmitters(final ApplicationStartupEvent event) {
+    try {
+      final Configs configs = new EnvConfigs();
```

Comment:
We should probably add a temporary note as to why we are creating all this stuff in here manually instead of allowing Micronaut to manage these objects.  That should protect future us from coming back in here and moving stuff to Micronaut management when we forget.

Reply:
@jdpgrailsdev Done